src/omp/exec_dist.f90

```diff
@@ -72,7 +72,7 @@ subroutine exec_dist_transeq_compact( &
     implicit none
 
     ! du = d(u)
-    real(dp), dimension(:, :, :), intent(out) :: rhs, du, dud, d2u
+    real(dp), dimension(:, :, :), intent(inout) :: rhs_du, dud, d2u
```

Comment:
From what I understand `dud` and `d2u` are still `intent(out)` only.

Reply:
They're actually all `intent(out)` as before, fixed.